phpunit.xml

```diff
@@ -7,19 +7,18 @@
          convertNoticesToExceptions="true"
          convertWarningsToExceptions="true"
          processIsolation="false"
-         stopOnFailure="false"
-         syntaxCheck="false">
-    <logging>
-        <log type="coverage-clover" target="build/logs/clover.xml"/>
-    </logging>
-    <testsuites>
-        <testsuite name="Test suite">
-            <directory suffix="Test.php">./tests/php/</directory>
-        </testsuite>
-    </testsuites>
-    <filter>
-        <whitelist>
-            <directory>./src</directory>
-        </whitelist>
-    </filter>
-</phpunit>
\ No newline at end of file
+         stopOnFailure="false">
```

Comment:
Please undo, the indents are now double

Reply:
Not sure if I understood you correctly, but should be fixed now